MapleServer2/Database/DatabaseContext.cs

```diff
@@ -348,8 +349,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                    i => i == null ? new MusicScore() : JsonConvert.DeserializeObject<MusicScore>(i));
 
                 entity.Property(e => e.Stats).HasConversion(
-                    i => JsonConvert.SerializeObject(i, settings),
-                    i => i == null ? new ItemStats() : JsonConvert.DeserializeObject<ItemStats>(i, settings));
```

Comment:
If the settings are not needed anymore, you can remove line 57.

Reply:
i'm dumb nvm it's still needed